### PR TITLE
Upgrade to rails5 beta3

### PR DESCRIPTION
**Pull request by jvanbaarsen**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -43,7 +45,7 @@ group :development, :test do
 end
 
 group :test do
-  gem "minitest-rails-capybara"
+  # gem "minitest-rails-capybara"
```
Comment (joshuajansen) [inline Gemfile]: You can remove this gem :)

File: config/routes.rb [hunk 2]
```diff
@@ -1,10 +1,6 @@
-require "sidekiq/web"
-require 'sidekiq/cron/web'
 require "user_constraint"
 
 Rails.application.routes.draw do
-  mount Sidekiq::Web => "/sidekiq", constraints: UserConstraint.new
```
Comment (joshuajansen) [inline config/routes.rb]: If you use `sinatra` from master, you'll be able to keep sidekiq web in :smile:

File: Gemfile (hunk 1 above)
Comment (jvanbaarsen) [inline Gemfile]: Uh yes I can 💃 And I will 😸

File: config/routes.rb (hunk 2 above)
Comment (jvanbaarsen) [inline config/routes.rb]: Interesting :-) Let me five that a try (Not that keen on using master though, will fork for now)

**Pull request merged**